### PR TITLE
feat: list api enpoints

### PR DESCRIPTION
**Pull request by Okhelifi**
## Issue

https://github.com/gravitee-io/issues/issues/8141

## Description

Migrate api proxy endpoints list page
Only the list is migrated. So every click will make you navigate into old screens. ( edit, view a group or an  endpoint )

### K8s
Screenshot 2022-10-10 at 11 26 42

### Management
Screenshot 2022-10-10 at 11 26 35

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/feat-8141-read-only-endpoints/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-mnooxyiqkd.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/api/proxy/endpoints/list/api-proxy-endpoint-list.component.ts [hunk 1]
```diff
@@ -105,6 +105,40 @@ export class ApiProxyEndpointListComponent implements OnInit {
...
+          _.remove(_.find(api.proxy.groups, (g) => g.name === groupName).endpoints, (e) => e.name === endpointName);
+          return this.apiService.update(api);
+        }),
+        catchError(({ error }) => {
+          this.snackBarService.error(error.message);
+          return EMPTY;
+        }),
+        tap((api) => {
+          this.initData(api);
+          this.ajsRootScope.$broadcast('apiChangeSuccess', { api: _.cloneDeep(api) });
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/management/api/proxy/endpoints/list/api-proxy-endpoint-list.component.ts]: not necessary if is all-ready in the service ? 🤔 or not ?

File: gravitee-apim-console-webui/src/management/api/proxy/endpoints/list/api-proxy-endpoint-list.component.html [hunk 2]
```diff
@@ -62,31 +62,39 @@ <h2>{{ endpointGroupData.name }}</h2>
       <!-- Name Column -->
       <ng-container matColumnDef="name">
         <th mat-header-cell *matHeaderCellDef id="name" width="20%">Name</th>
-        <td mat-cell *matCellDef="let element">
+        <td mat-cell [class.endpoint-list__table__backup-row]="element.isBackup" *matCellDef="let element">
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/management/api/proxy/endpoints/list/api-proxy-endpoint-list.component.html]: wdyt to add a badge istead a color change ? like [secondary] with 
image

**Review by ThibaudAV — APPROVED** [review]

File: gravitee-apim-console-webui/src/management/api/proxy/endpoints/list/api-proxy-endpoint-list.component.ts (hunk 1 above)
Comment (Okhelifi) [reply gravitee-apim-console-webui/src/management/api/proxy/endpoints/list/api-proxy-endpoint-list.component.ts]: I think it's outdated. There is a dedicated commit where I remove all calls to the root scope normally.

File: gravitee-apim-console-webui/src/management/api/proxy/endpoints/list/api-proxy-endpoint-list.component.html (hunk 2 above)
Comment (Okhelifi) [reply gravitee-apim-console-webui/src/management/api/proxy/endpoints/list/api-proxy-endpoint-list.component.html]: Here it is. You're right it's better like that I think 👍🏼 

Screenshot 2022-10-11 at 09 23 29

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**